gui/src/components/MainLayout.scss

```diff
@@ -0,0 +1,24 @@
+.main-layout {
+    display: grid;
```

Comment:
Why is this not in tailwind?

Reply:
i think it makes sense in this case, because it needs to be a grid for the grid template to be applied